src/app/help/page.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import UnderDevelopment from '@/components/general/UnderDevelopment/UnderDevelopment';
+import { monotonFont } from '@/helpers/font'
+import { BackgroundBeams } from '@/components/ui/background-beams';
+import React from 'react'
+import { motion } from 'motion/react';
+
+const page = () => {
```

Comment:
Need to change the component name for better readability.


```suggestion
const HelpPage = () => {
```